EDDI/EDDI.cs

```diff
@@ -106,6 +107,11 @@ public static EDDI Instance
         public Station HomeStation { get; private set; }
         public StarSystem SquadronStarSystem { get; private set; } = new StarSystem();
 
+        // Destination variables
+        public StarSystem DestinationStarSystem { get; private set; }
+        public Station DestinationStation { get; private set; }
+        public decimal DestinationDistance { get; set; }
```

Comment:
`DestinationDistanceLy`? (to include units in the name)

Reply:
Sounds reasonable.

Reply:
Changed.